modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -36,36 +36,88 @@ Establishing the right performance expectations refers to the fact that the real
 . Install Performance Addon Operator to the cluster.
 . Optional: Add a node to the {product-title} cluster.
 See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/html-single/tuning_guide/index#Setting_BIOS_parameters[Setting BIOS parameters].
-. Optional: Create a new machine config pool for real-time nodes.
+. Create a new machine config pool for real-time nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-cnf
+  labels:
+    machineconfiguration.openshift.io/role: worker-cnf
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+           key: machineconfiguration.openshift.io/role,
+           operator: In,
+           values: [worker, worker-cnf],
```

Comment:
as above, use worker-rt (instead of worker-cnf) or we use worker-cnf in all places.